backend-db/modules/db/couchdb/index.js

```diff
@@ -402,12 +403,23 @@ const layer = {
         create.hashtags = create.hashtags || [];
 
         const id = util.uuid.random();
+        const account = await layer.getAccount(who);
```

Comment:
There's no need to fetch the account. Since `who` is the currently logged-in account, all its information is available in the microservice sideload data (`__` parameter in the Seneca plugin that called `db.createTrashpoint()`). So `team` can be taken from `__` and passed as parameter to this function.